core/src/scss/components/main-nav/_main-nav--dark.scss

```diff
@@ -1,128 +1,137 @@
 @charset "UTF-8";
 
-.su-main-nav {
-  &.su-main-nav--dark {
-    @include grid-media-max('md') {
-      .su-main-nav__toggle {
-        color: $color-white;
-
-        &:hover,
-        &:focus {
-          &::before {
-            background-color: $color-white;
-          }
+.su-main-nav--dark {
+  @include grid-media-max('md') {
+    .su-main-nav__toggle {
+      color: color(link, $main-nav-colors, dark);
+
+      &:hover,
+      &:focus {
+        &::before {
+          background-color: color(border--hover, $main-nav-colors, dark);
```

Comment:
This is a little less clear than the two parameter function. Instead of having nested themes, why wouldn't you just pass in a different color map? eg:

`background-color: color(border--hover, $main-nav-colors-dark);`